airsonic-main/src/main/java/org/airsonic/player/Application.java

```diff
@@ -1,185 +1,79 @@
-        "classpath:/applicationContext-cache.xml",
-        "classpath:/applicationContext-sonos.xml",
-        "classpath:/servlet.xml"})
-public class Application extends SpringBootServletInitializer implements EmbeddedServletContainerCustomizer {
+        MultipartAutoConfiguration.class // TODO: update to use spring boot builtin multipart support
+})
+public class Application extends SpringBootServletInitializer
+        implements WebServerFactoryCustomizer<ConfigurableServletWebServerFactory> {
 
     private static final Logger LOG = LoggerFactory.getLogger(Application.class);
 
-    /**
-     * Registers the DWR servlet.
-     *
-     * @return a registration bean.
-     */
-    @Bean
-    public ServletRegistrationBean dwrServletRegistrationBean() {
-        ServletRegistrationBean servlet = new ServletRegistrationBean(new DwrServlet(), "/dwr/*");
-        servlet.addInitParameter("crossDomainSessionSecurity","false");
-        return servlet;
-    }
-
-    @Bean
-    public ServletRegistrationBean cxfServletBean() {
-        return new ServletRegistrationBean(new org.apache.cxf.transport.servlet.CXFServlet(), "/ws/*");
-    }
-
-    @Bean
-    public ServletContextListener ehCacheShutdownListener() {
-        return new ShutdownListener();
-    }
-
-    @Bean
-    public FilterRegistrationBean bootstrapVerificationFilterRegistration() {
-        FilterRegistrationBean registration = new FilterRegistrationBean();
-        registration.setFilter(bootstrapVerificationFiler());
-        registration.addUrlPatterns("/*");
-        registration.setName("BootstrapVerificationFilter");
-        registration.setOrder(1);
-        return registration;
-    }
-
-    @Bean
-    public Filter bootstrapVerificationFiler() {
-        return new BootstrapVerificationFilter();
-    }
-
-    @Bean
-    public FilterRegistrationBean parameterDecodingFilterRegistration() {
-        FilterRegistrationBean registration = new FilterRegistrationBean();
-        registration.setFilter(parameterDecodingFilter());
-        registration.addUrlPatterns("/*");
-        registration.setName("ParameterDecodingFilter");
-        registration.setOrder(2);
-        return registration;
-    }
-
-    @Bean
-    public Filter parameterDecodingFilter() {
-        return new ParameterDecodingFilter();
-    }
-
-    @Bean
-    public FilterRegistrationBean restFilterRegistration() {
-        FilterRegistrationBean registration = new FilterRegistrationBean();
-        registration.setFilter(restFilter());
-        registration.addUrlPatterns("/rest/*");
-        registration.setName("RESTFilter");
-        registration.setOrder(3);
-        return registration;
-    }
+    private static SpringApplicationBuilder doConfigure(SpringApplicationBuilder application) {
+        // Customize the application or call application.sources(...) to add sources
+        // Since our example is itself a @Configuration class (via @SpringBootApplication)
```

Comment:
Our example?

Reply:
Yes, "our example"

This comment is not from me. It is part of the existing class. If you search for the doConfigure method in the existing class, you'll see that comment.